src/keri/vc/proving.py

```diff
@@ -4,52 +4,51 @@
 
 """
 
-from collections.abc import Iterable
-from typing import Union
-
-from .. import help
+from typing import Optional, Union
 
 from .. import core
+from .. import help
 from ..core import coring, serdering
 from ..core.coring import (Kinds, versify)
-from ..db import subing
-from ..kering import Version
 from ..help import helping
+from ..kering import Version
 
 KERI_REGISTRY_TYPE = "KERICredentialRegistry"
 
 logger = help.ogler.getLogger()
 
 
-def credential(schema,
-               issuer,
-               data,
-               recipient=None,
-               private=False,
-               salt=None,
-               status=None,
-               source=None,
-               rules=None,
-               version=Version,
-               kind=Kinds.json):
+def credential(schema:str,
```

Comment:
Thank you for fixing the type hints in the PR too, I'm surprised I didn't have to remind you here as well...  nicely done.